pkg/manifests/manifests.go

```diff
@@ -2878,6 +2878,39 @@ func (f *Factory) ControlPlanePrometheusRule() (*monv1.PrometheusRule, error) {
 		}
 		r.Spec.Groups = groups
 	}
+	if !f.infrastructure.HighlyAvailableInfrastructure() {
+		// We are in SNO. Aggregated API may go down at times since there is no high
+		// availability for these components. Examples of these situations may
+		// include upgrades or certificate rotations. To try to avoid expected alerts
+		// we change the KubeAggregatedAPIDown severity to info, as its not an anomaly,
+		// and introduce a new alert KubeAggregatedAPIOffline, which fires over longer
+		// window times to detect downtime when its not expected.
+		for i := range r.Spec.Groups {
+			group := &r.Spec.Groups[i]
+			if group.Name == "kubernetes-system-apiserver" {
+				for j := range group.Rules {
+					rule := &group.Rules[j]
+					if rule.Alert == "KubeAggregatedAPIDown" {
+						rule.Labels["severity"] = "info"
+						break
+					}
+				}
+				group.Rules = append(group.Rules, monv1.Rule{
+					Alert: "KubeAggregatedAPIUnavailable",
```

Comment:
```suggestion
					Alert: "KubeAggregatedAPIOffline",
```

Reply:
Another option would be keep the existing `KubeAggregatedAPIDown` but extend the for duration in case of SNO.

Reply:
@omertuc Done.
@simonpasquier Would that be acceptable for a multi node deployment? We would be changing the behavior of this alert for already existing releases. Problem with SNO is that downtime is expected, so you know it's going to happen and that's normal behavior. In multi node this is not the case.

Reply:
@simonpasquier misunderstood, I actually like this idea better. Thanks!